src/main/java/budgetflow/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package budgetflow.command;
+
+import budgetflow.exception.FinanceException;
+import budgetflow.expense.ExpenseList;
+import budgetflow.income.Income;
+
+import java.util.List;
+
+public class Command {
+    protected CommandType commandType;
+    protected String input;
+    protected String outputMessage;
+    protected boolean isExit = false;
+
+    public Command() {}
+
+    public Command(String input) {
+        this.input = input;
+    }
+
+    public void execute(List<Income> incomes, ExpenseList expenseList) throws FinanceException {
+        throw new FinanceException("This method is operated by child classes");
```

Comment:
Error message is a needs to be clearer and more specific. If this is relayed to the user then it will be confusing to them (they will not know what "method" your application is referring to).